CONTRIBUTING.org

```diff
@@ -0,0 +1,213 @@
+* Contributing a new recipe to MELPA
+
+MELPA consists of /recipes/, each describing one /package/ in a
+dedicated repository. This document describes how to propose a new
+package for inclusion into MELPA.
+
+Three contributor roles are involved in the maintenance of MELPA :
+
+- MELPA maintainers :: They are responsible for reviewing new recipes
+     and associated packages as well as keeping the system working.
+
+- Recipe authors :: They responsible for submitting good recipes and
+                    maintaining them when the package is changed
+                    (e.g., renamed or deprecated).
+
+- Package author :: They are responsible for writing Emacs Lisp
+                    packages of good quality.
+
+We advise package author to also take the recipe author role when
+possible.
+
+In the following, package and recipe authors will find guidelines to
+help them prepare a package and its recipe for inclusion into
+MELPA. Please follow the guidelines as closely as possible to speed up
+the inclusion process. Guidelines are not strict rules though and they
```

Comment:
“Guidelines are not strict and evaluated on a case by case basis given proper justification.“